Feature/10 Ignoreing directories (#11)
exasol/artifact-reference-checker-maven-plugin: README.md

* Run checks manually: `mvn artifact-reference-checker:verify`
* Unify the versions using: `mvn artifact-reference-checker:unify`

## Configuration

* Exclude file:
    You can exclude files from verification and unification.
     
     ```xml
    <excludes>
        <exclude>doc/changes/*</exclude>
    </excludes>
    ``` 
  

>   If the path starts with a `/` it is relative to the projects root directory. Else it matches in all project folders.

Reviewer: ```suggestion
  If the path starts with a `/` it is relative to the project's root directory. Otherwise, it excludes path from project folders.
```

Author: ```suggestion
  If the path starts with a `/` it is relative to the project's root directory. Otherwise, it acts as a pattern that can also exclude files in nested directories without specifying the whole path. For example `*.md`.
```

Reviewer: Much better!
